### DISCUSSION TITLE
HPA

### DISCUSSION THREAD

**Discussion by mineo-reitz** [airflow-helm/charts#881] [Questions & Answers]
Hi everyone. I am using Airflow 2.8.4 with Helm 8.9.0. I have configured the HPA (Horizontal Pod Autoscaler), setting resources for Git-Sync and Workers.

Worker resources:
![image](https://github.com/user-attachments/assets/0080a309-a86d-4b84-8c8e-da8682144605)
_Note: I tried to lower the resources and got "OOMKilled"_

Git-Sync resources:
![image](https://github.com/user-attachments/assets/756ddee5-a560-4d78-9413-a14ecd1e3c3f)

This is my DEV EKS Cluster, and even without any DAGs running, the HPA spins up all the available pods, reaching the MaxReplica limit, which is 6.

![image](https://github.com/user-attachments/assets/49895c61-0a93-4300-a8b3-6dc5559be50e)

![image](https://github.com/user-attachments/assets/2a0034db-66eb-49f0-842a-d432422e6243)

Thank you in advance, any help will be appreciated

**Comment by pre-sumitbhardwaj** [comment]
@mineo-reitz have you got any solution for that, as I am also facing same issue?
